airflow/settings.py

```diff
@@ -79,18 +84,13 @@
 json = json
 
 
-def policy(task):
+def policy(task: 'BaseOperator'):
     """
-    This policy setting allows altering tasks right before they
-    are executed. It allows administrator to rewire some task parameters.
-
-    Note that the ``Task`` object has a reference to the DAG
-    object. So you can use the attributes of all of these to define your
-    policy.
+    This policy setting allows altering tasks after they are loaded in
+    the DagBag. It allows administrator to rewire some task parameters.
 
     To define policy, add a ``airflow_local_settings`` module
-    to your PYTHONPATH that defines this ``policy`` function. It receives
-    a ``Task`` object and can alter it where needed.
```

Comment:
Did you mean to remove these doc  comments? This bit seems appropriate still.

Reply:
I moved that to the sections below.